examples/small-study/src/study.js

```diff
@@ -0,0 +1,144 @@
+// put the config in the scope so that background can see it.
+const STUDYCONFIG = (this.STUDYCONFIG = {
```

Comment:
This variable should be called studySetup as long as the corresponding schema is called studySetup (which it is currently)